### PR TITLE
Mention `copy_paste` applies only when using segmentation labels

### LINKED ISSUE #18073: how to use  copy-paste for   object detection tasks
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I have a question regarding the use of the copy-paste method. Although YOLO has a new implementation, when I debug, I find that the segments in the labels are empty. To use this method, do I need to add additional mask data? I have experimented with adding and not adding the mask, but the results seem to be the same. Could you please teach me how to effectively use the copy-paste data augmentation method in YOLO for object detection tasks? Thank you so much.

### Additional

_No response_

### PR DESCRIPTION
**Pull request by Y-T-G**
Closes #18073

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary  
Updated documentation for the `copy_paste` augmentation, clarifying its functionality and requirements. ✍️

### 📊 Key Changes  
- Enhanced the description of the `copy_paste` augmentation to specify that it **requires segmentation labels** for proper functionality.

### 🎯 Purpose & Impact  
- **Purpose**: To provide clear, accurate documentation for users leveraging `copy_paste`, ensuring they understand its prerequisites.  
- **Impact**: Reduces confusion for users and improves the user experience 🌟, especially aiding those working on object detection/segmentation tasks.

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Y-T-G, thank you for submitting an `ultralytics/ultralytics` 🚀 PR! To ensure a smooth review and merge process, please review the following checklist:

- ✅ **Define a Purpose**: Your PR description is clear and links to the relevant issue. Thank you for ensuring clarity around the `copy_paste` augmentation feature! 
- ✅ **Synchronize with Source**: Ensure your PR is synchronized with the `ultralytics/ultralytics` `main` branch. If it's behind, you can update it by clicking the 'Update branch' button or by running `git pull` and `git merge main` locally.
- ✅ **Ensure CI Checks Pass**: Verify all Ultralytics [Continuous Integration (CI)](https://docs.ultralytics.com/help/CI/) checks pass. If there are any failures, please address them before requesting a review.
- ✅ **Update Documentation**: Your update to the documentation looks fantastic; users will definitely benefit from this added detail. 
- ✅ **Add Tests**: Confirm or add/update any relevant tests to cover this update, ensuring no disruptions or unexpected behavior.
- ✅ **Sign the CLA**: Please ensure that you’ve signed our [Contributor License Agreement (CLA)](https://docs.ultralytics.com/help/CLA/) by commenting "I have read the CLA Document and I sign the CLA" in this thread if this is your first contribution.
- ✅ **Minimize Changes**: Your changes appear focused and targeted, which we highly appreciate! Keeping updates concise ensures a seamless integration process.

For additional information on contributing to our project, feel free to consult the [Contributing Guide](https://docs.ultralytics.com/help/contributing).

✨ This is an automated response to help streamline the PR workflow. An Ultralytics engineer will review your PR and assist further shortly. If you have any questions in the meantime, don't hesitate to ask. Thank you for making Ultralytics even better! 🚀

File: docs/en/macros/augmentation-args.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 | `bgr`             | `float` | `0.0`         | `0.0 - 1.0`   | Flips the image channels from RGB to BGR with the specified probability, useful for increasing robustness to incorrect channel ordering.                                  |
 | `mosaic`          | `float` | `1.0`         | `0.0 - 1.0`   | Combines four training images into one, simulating different scene compositions and object interactions. Highly effective for complex scene understanding.                |
 | `mixup`           | `float` | `0.0`         | `0.0 - 1.0`   | Blends two images and their labels, creating a composite image. Enhances the model's ability to generalize by introducing label noise and visual variability.             |
-| `copy_paste`      | `float` | `0.0`         | `0.0 - 1.0`   | Copies objects from one image and pastes them onto another, useful for increasing object instances and learning object occlusion.                                         |
+| `copy_paste`      | `float` | `0.0`         | `0.0 - 1.0`   | Copies and pastes objects across images, useful for increasing object instances and learning object occlusion. Applies only to segmentation training.                     |
```
Comment (Laughing-q) [inline docs/en/macros/augmentation-args.md:16]: it's not segmentation training only, we could use it for object detection but it does need segment labels.

Comment (Y-T-G) [reply docs/en/macros/augmentation-args.md]: @Laughing-q Ah ok. That makes sense.

**Review by UltralyticsAssistant — APPROVED** [review]

**Comment by glenn-jocher** [comment]
Merged!

**Pull request merged**